config/.shield.sample.yaml

```diff
@@ -0,0 +1,52 @@
+version: 1
```

Comment:
@mabdh let's rename this to just `config.yaml` (without sample word) since we use `shield.yaml` for client config.  wdyt?

Reply:
yes makes sense, updating it